### PR TITLE
Completed Feature 3: User can view review

### PR DESCRIPTION
**Pull request by ellie-hwang**
view review mobile 01
view review mobile 02
view review desktop

### CODE REVIEW

File: js/main.js [hunk 1]
```diff
@@ -321,8 +347,80 @@ function createReview(event) {
...
+  $divEl3.className = 'pad-1';
+  var $h3El = document.createElement('h3');
+  $h3El.className = 'no-marg review-title';
+  $h3El.textContent = reviewObj.title;
+  var $pEl = document.createElement('p');
+  $pEl.className = 'review-notes pad-2-bot';
+  $pEl.textContent = reviewObj.reviewNotes;
+  // var $divEl4 = document.createElement('div');
+  // var $iEl6 = document.createElement('i');
+  // $iEl6.className = 'fas fa-pen-alt pen-icon';
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code prior to submitting for review

File: js/main.js [hunk 2]
```diff
@@ -321,8 +347,80 @@ function createReview(event) {
...
+
+  $divEl1.appendChild($imgEl);
+  $divEl3.appendChild($h3El);
+  $divEl3.appendChild($pEl);
+  // $divEl4.appendChild($iEl6);
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code prior to submitting for review

File: js/main.js [hunk 3]
```diff
@@ -321,8 +347,80 @@ function createReview(event) {
...
+  $liEl.appendChild($divEl1);
+  $liEl.appendChild($divEl2);
+  $liEl.appendChild($divEl3);
+  // $liEl.appendChild($divEl4);
```
Comment (codingcodymiller) [inline js/main.js]: Please remove commented code prior to submitting for review

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

File: js/main.js (hunk 3 above)
Comment (ellie-hwang) [reply js/main.js]: Removed!

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**